.travis.yml

```diff
@@ -1,12 +1,17 @@
 language: python
 python:
-    - 2.6
     - 2.7
     - 3.3
```

Comment:
Perhaps it's time to drop 3.3 as well?  I think large parts of the ecosystem (setuptools?) no longer support it.

Reply:
Done.